MNE/rtProcessing/rtave.cpp

```diff
@@ -302,14 +302,25 @@ void RtAve::run()
 
                     m_bFillingBackBuffer = false;
 
-                    qDebug()<<"Number of calculated averages:"<<m_iNumberCalcAverages;
-                } else
+                    qDebug()<<"RtAve::run() - Number of calculated averages:"<<m_iNumberCalcAverages;
```

Comment:
debug necessary - we should think about a logger class - since qDebug() could lead to a speed issue